modules/home-manager/waybar.nix

```diff
@@ -0,0 +1,40 @@
+{ config
+, lib
+, sources
+, ...
+}:
+let
+  cfg = config.programs.waybar.catppuccin;
+  enable = cfg.enable && config.programs.waybar.enable;
+in
+{
+  options.programs.waybar.catppuccin = (lib.ctp.mkCatppuccinOpt "waybar") // {
+    styleFile = lib.mkOption {
+      type = lib.types.path;
+      default = "${sources.waybar}/themes/${cfg.flavour}.css";
+      description = "Path to CSS file containing color variable definitions";
+    };
```

Comment:
im a bit hesitant on allowing this to be an option. no other module has this, and i feel like removing the file from the port would kinda defeat the purpose of using our module in the first place - as without it, you don't have a catppuccin theme. using alternative versions of this file could also be covered under https://github.com/catppuccin/nix/pull/129

Reply:
Yes, #129 definitely is the better solution once it hits. I removed the option for now.